smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/validation/PropertyAccessor.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.smithy.typescript.codegen.validation;
+
+import java.util.regex.Pattern;
+
+/**
+ * TODO: deprecate and remove the copy of this utility in JS SDK V3 codegen.
+ */
+public final class PropertyAccessor {
```

Comment:
Maybe you missed some commits? I cannot find this class is refered any where.

Reply:
I'm moving it here in anticipation of calling it later. There is a duplicate in V3.

Reply:
Can you share where it is called in v3? I feel like this change belongs to a separate PR

Reply:
ok, I agree and I'll move it out